scripts/exregional_aqm_ics.sh_orig

```diff
@@ -0,0 +1,161 @@
+#!/bin/bash
```

Comment:
@JianpingHuang-NOAA, if this script and the above script (*.issue926) are not necessary, please remove them.

Reply:
Deleted.